trino-aws-proxy/src/test/java/io/trino/aws/proxy/server/testing/TestingTrinoAwsProxyServer.java

```diff
@@ -222,6 +223,9 @@ public TestingTrinoAwsProxyServer buildAndStart()
                 modules.add(binder -> binder.bind(Credentials.class).annotatedWith(ForTestingRemoteCredentials.class).toProvider(TestingRemoteCredentialsProvider.class));
             }
 
+            addModule(remoteS3Module("testing", TestingRemoteS3Facade.class, (binder) -> binder.bind(TestingRemoteS3Facade.class).in(Scopes.SINGLETON)));
```

Comment:
With the change of binding the default directly (not as a plugin) we can get rid of TestingRemoteS3Facade completely I think. Tests that need a custom RemoteS3Facade can now install a plugin and set the config

Reply:
It is possible, but it would require some changes in how test environment is bootstrapped - to replace implementations from `ContainerS3Facade` with configuring properties for remote S3 host and port - but currently `S3Container` is initialized by guice, so we don't have the host and port when configuring `TestingTrinoAwsProxyServer`.

Reply:
That is a fair point, it would be nice to have but it would be too much pain at this point to add. 

I would say that line 226 and 227 need to be moved into `withS3Container`. Otherwise it is impossible to setup a custom `remoteS3Module` since `withProperty("remote-s3.type", "testing")` overrides what other BuilderFilters might be settings

Reply:
That's a good catch! Fixed.